chapters/clears.txt

```diff
@@ -162,16 +164,8 @@ endif::VK_VERSION_1_1,VK_KHR_maintenance1[]
     pname:imageLayout must: be either of
     ename:VK_IMAGE_LAYOUT_TRANSFER_DST_OPTIMAL or
     ename:VK_IMAGE_LAYOUT_GENERAL
-  * [[VUID-vkCmdClearDepthStencilImage-aspectMask-02499]]
-    The slink:VkImageSubresourceRange::pname:aspectMask members of the
-    elements of the pname:pRanges array must: each only include
-    ename:VK_IMAGE_ASPECT_DEPTH_BIT if the image format has a depth
-    component
-  * [[VUID-vkCmdClearDepthStencilImage-aspectMask-02500]]
-    The slink:VkImageSubresourceRange::pname:aspectMask members of the
-    elements of the pname:pRanges array must: each only include
-    ename:VK_IMAGE_ASPECT_STENCIL_BIT if the image format has a stencil
-    component
```

Comment:
So I think this is a case of unclear english language usage - it's meant to be like "only include these ASPECT bits if there's the right component" as opposed to "include only this ASPECT bit if the component matches". Language.

I think this can be clarified by just flipping the existing text condition around, so that it says "must: not include <X> if the image format does not have a <Y> component. I.e.:

```
  * [[VUID-vkCmdClearDepthStencilImage-aspectMask-02499]]
    The slink:VkImageSubresourceRange::pname:aspectMask members of the
    elements of the pname:pRanges array must: not include
    ename:VK_IMAGE_ASPECT_DEPTH_BIT if the image format does not have a
    depth component
  * [[VUID-vkCmdClearDepthStencilImage-aspectMask-02500]]
    The slink:VkImageSubresourceRange::pname:aspectMask members of the
    elements of the pname:pRanges array must: not include
    ename:VK_IMAGE_ASPECT_STENCIL_BIT if the image format does not have a
    stencil component
```

This retains the original meaning/VUIDs/precision of the existing VUs.

Reply:
Hmm, what about all the other aspects?

Reply:
This command is already restricted to depth/stencil formats, which takes care of the rest.

Reply:
yes, but not to aspects.

Reply:
Oh I see. I think then to remain precise we just add:

```
  * slink:VkImageSubresourceRange::pname:aspectMask must: not include bits other than ename:VK_IMAGE_ASPECT_STENCIL_BIT or ename:VK_IMAGE_ASPECT_DEPTH_BIT.
```

Reply:
Are you sure none of these new aspects apply here? I am not familiar with them much...

Reply:
Right, if you prefer.

Reply:
done